acorn/src/lval.js

```diff
@@ -158,14 +158,18 @@ pp.parseBindingList = function(close, allowEmpty, allowTrailingComma) {
       this.expect(close)
       break
     } else {
-      let elem = this.parseMaybeDefault(this.start, this.startLoc)
-      this.parseBindingListItem(elem)
-      elts.push(elem)
+      elts.push(this.parseAssignableListItem(allowModifiers))
     }
   }
   return elts
 }
 
+pp.parseAssignableListItem = function(allowModifiers) {
```

Comment:
Please call this `parseBindingListItem` for consistency.

Reply:
There is a method named `parseBindingListItem`. How to solve this question >n<?
```
pp.parseBindingListItem = function(param) {
  return param
}
```

Reply:
Indeed, that's true. What is the reason you need another wrapper, and can't override that instead?

Reply:
Yes, I cant override the `parseBindingListItem` instead. I have to override the logic of this part. Do something before calling the method `parseMaybeDefault`.
```
pp.parseAssignableListItem = function() {
  let elem = this.parseMaybeDefault(this.start, this.startLoc)
  this.parseBindingListItem(elem)
  return elem
}
```

Reply:
> Indeed, that's true. What is the reason you need another wrapper, and can't override that instead?

Because Ts support the Parameter Decorators. Is it reasonable to add a hook while parsing the items ?

Reply:
@marijnh Hello, Could u checkout this question, plz.